pkg/config/autodiscover/autodiscover_targets.go

```diff
@@ -19,7 +19,11 @@ package autodiscover
 import (
 	log "github.com/sirupsen/logrus"
 	"github.com/test-network-function/test-network-function/pkg/config/configsections"
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	"github.com/test-network-function/test-network-function/pkg/tnf/handlers/nodenames"
+	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
 	"github.com/test-network-function/test-network-function/pkg/tnf/testcases"
+	"github.com/test-network-function/test-network-function/test-network-function/common"
```

Comment:
@jun-rh  
I believe the reason why they are using direct 'oc' command in autodiscover package is to avoid cyclic dependency. 
with this change, autodiscover package depends on common package. and it should not be the case

We should move this function : 
```
func GetContext() *interactive.Context {
```
from common package and move it into pkg/interactive/spawner
once this is done, we can stop using "oc" directly without the context

Reply:
I can make this change later in my PR. so don't worry about it.

Reply:
Not sure if it really was the reason. Before I centralized all the spawn local shell calls to common.GetContext(), the contexts were created in many different ways. Some were just a spawn shell call. Some have one or two assertions after. Some suites had a local getContext() function with 3 assertions. If they were to use handler in autodiscover, they probably wouldn't be afraid of repeating a few lines there. Anyway, I am not sure if we want to use gomega in the pkg/tnf/interactive. Right now it's only used in pkg/config and pkg/tnf/test.go

Reply:
right now, with this change, the autodiscovery code will depend on "common" which in itself depends on gomega. The autodiscovery code should not depend on the "common" code. and we should create our own session instead of getting it from common.